content/post/2020-10-14-dv-signal.Rmd

```diff
@@ -0,0 +1,259 @@
+
+<!--more-->
+
+In previous blog posts, we have written about indicators generated from surveys that we run through [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Google](https://delphi.cmu.edu/blog/2020/09/18/covid-19-symptom-surveys-through-google/). In addition to these, we have also generated indicators from electronic medical records. We received access to hospital records and insurance claims data from healthcare partners covering 10-15% of the population of the United States. One indicator that we created from this data is the "Doctor Visits" signal, which estimates the percentage of office visits due to COVID-like illness.
+
+Electronic health records are extremely useful in providing information about epidemics in real time. A patient may interact with the healthcare system days before receiving a test (if any), and several more days before receiving a positive result and being reported as a confirmed case by a state's Department of Health. This can make our signal an **early indicator** of confirmed cases, and certainly an early indicator of deaths. Early knowledge of an increase in people with COVID-like symptoms is extremely important, as early containment can prevent an outbreak.
+
+Additionally, the reported case counts have other issues:
+
+* Limited testing capacity: Especially prevalent in the earlier days of the pandemic, limited testing capacity means that there were many people who saw a doctor for COVID-like symptoms but never even received a test. These people would be present in our data but absent from the confirmed counts.
+* Backlogs: A patient can be first reported as a confirmed case can be several days after they were tested. Since our data is linked to the date that the patient interacted with the healthcare system, we report a more accurate date.
+* Revisions: Some states have changed their definition of which tests indicate a confirmed COVID-19 case. This can be done retroactively, including hundreds or thousands of past cases and reporting them as "new" on the day on which they changed their definition. These cases are clearly assigned the wrong date, but we have no way of determining when the test was actually taken. Our data is subject to revisions due to lags in reporting, but the revisions are understood and are assigned the correct date.
```

Comment:
```suggestion
* Limited testing capacity: Especially prevalent in the earlier days of the pandemic, limited testing capacity means that there were many people who saw a doctor for COVID-like symptoms but never received a test even though they should have. These people would be present in our data but absent from the confirmed counts.
* Reporting delays:  From the moment a patient is first seen or evaluated, it may take several days (especially in the early days of the pandemic) until their specimen is collected.  It then takes a few more days for the test to be run.  Next, if the test is positive it is reported to the health authorities as required by law.  The health authorities typically do further verification and investigation (for example, determining age and other demographics) before adding the case into their public website.  As a result, many patients are reported as confirmed cases several weeks after they were first medically attended.  While insurance claims also have built-in delays (see below), a meaningful claims-based signal can be estimated within 3-4 days of the doctor's visit.
* Dating accuracy: Cumulative confirmed cases scraped and reported by sites such as JHU and USA Facts (and mirrored by COVIDcast) are based on the date the case counts were publicly posted.  As a result, the "number of new daily counts" derived by successive subtraction from these reports refers to the number of cases _published_ on that date, not those that were _lab-confirmed_ on that date.  In contrast, insurance claims records indicate the "date of service" -- the date that the patient interacted with the healthcare system, which is typically closer to the testing date.  As explained above, there can be a gap of up to several weeks between these two dates.
* Retrospective revisions: Some states have changed their definition of confirmed COVID-19 case, sometimes. This is sometime done retroactively, adding hundreds or thousands of past cases and reporting them as "new" on the day on which the definition was changed. These cases are clearly assigned the wrong date, but we have no way of determining when the test was actually taken.  Insurance claims data is not subject to such revisions.
```